devtools/conda-envs/environment.yaml

```diff
@@ -0,0 +1,13 @@
+name: membrane-curvature
+channels:
+  - conda-forge
+  - defaults
+dependencies:
+  - python
+  - pip
+  - numpy
+  - MDAnalysisTests
+
+    # Pip-only installs
+  - pip:
+    - MDAnalysis==2.0.0b0
```

Comment:
Would `=>` work here? This would avoid having to immediately change this once 2.0 is properly out?

Reply:
Yes, added, thanks.